### DISCUSSION TITLE
A problem arise when discussing a pull request ?

### DISCUSSION THREAD

**Discussion by ManashAnand** [EddieHubCommunity/LinkFree#8266] [Q&A] [answered]
I want to talk to contributor and she replied but I am not able to move conversation ahead as github bot put an end or some limit to conversation .
![np](https://github.com/EddieHubCommunity/LinkFree/assets/91553087/71dd4a59-426a-4403-9ba1-bb1adf378293)
How to reach them and move the conversation ahead?

**Comment by eddiejaoude** [comment, marked as answer]
Oh sorry, we should unlock the issue after we leave a comment - I will unlock now

**Comment by ManashAnand** [comment]
It's okay ,
Thanks for solving my doubt
